pkg/iss/iss.go

```diff
@@ -70,9 +72,6 @@ type ISS struct {
 	// Implementation of the hash function to use by ISS for computing all hashes.
 	hashImpl crypto.HashImpl
 
-	// Verifier of received stable checkpoints. This is most likely going to be the crypto module used by the protocol.
```

Comment:
This is actually very nice. That we don't need to pollute the ISS module with the checkpoint verifier.

(Resolve this directly, not change requested :-) )